app/Http/Controllers/Teams/TeamsController.php

```diff
@@ -231,11 +211,11 @@ public function joined ()
      */
     public function members ()
     {
-        $query = $this->filterTeamMembers(request()->team_id);
+        $team = Team::query()->find(request()->team_id);
 
-        $total_members = $query->users->count(); // members?
+        $total_members = $team->users->count(); // members?
```

Comment:
I think the name `users` is wrong on the team model. It should have a keyword of members. Also, if we are getting the count dynamically, we should drop incrementing and decrementing. In my experience incr/decr rarely works as expected

Reply:
True, using `members` would make much more sense. Not sure why we need the `$total_members`, would probably be a good refactoring to just remove it, but we need to make sure it's not used.